src/js/edu-benefits/5495/config/form.js

```diff
@@ -104,6 +104,15 @@ const formConfig = {
               'view:noSSN': {
                 'ui:title': 'I don’t know my sponsor’s Social Security number',
               },
+              veteranSocialSecurityNumber: {
+                'ui:validations': [
+                  (errors, fieldData, pageData, formData) => {
+                    if (fieldData === formData.relativeSocialSecurityNumber) {
+                      errors.addError("Your sponsor's SSN cannot be the same as yours.");
```

Comment:
Lint didn't yell at me...Probably because I've got the single quote as an apostrophe for "sponsor's".

Fair enough.